README.md

```diff
@@ -0,0 +1,260 @@
+# theming
+
+[![NPM version][npm-image]][npm-url]
+[![Build][travis-image]][travis-url]
+[![Coveralls Status][coveralls-image]][coveralls-url]
+[![Dependency Status][depstat-image]][depstat-url]
+
+> React High-Order Components for theming
```

Comment:
I would add something like "A unifying solution for theming in React for CSSinJS"